lib/npm/install.js

```diff
@@ -18,6 +18,9 @@ function install(context, next) {
   if (context.options.nodedir) {
     var nodedir = path.resolve(process.cwd(), context.options.nodedir);
     options.env.npm_config_nodedir = nodedir;
+    if (process.platform === 'win32') {
+      nodedir = nodedir.replace(/\\/g, '\\\\');
```

Comment:
I was thrown off by `\\` being replace with `\\\\`.

I'll admit I don't know much about windows paths, but it seems like an extra `\`